Makefile

```diff
@@ -1,89 +1,97 @@
-ITEMS   := shared/html shared/data shared/img
-
-# Workaround Browserify not following symlinks in --only.
-# TODO: Remove this once Browserify has been replaced with a different bundler.
-BROWSERIFY_GLOBAL_TARGETS = ./node_modules/@duckduckgo
-BROWSERIFY_GLOBAL_TARGETS += $(shell find node_modules/@duckduckgo/ -maxdepth 1 -type l | xargs -n1 readlink -f)
-
-###--- Binaries ---###
-SASS = node_modules/.bin/sass
-BROWSERIFY_BIN = node_modules/.bin/browserify
-BROWSERIFY = $(BROWSERIFY_BIN) -t babelify -t [ babelify --global  --only [ $(BROWSERIFY_GLOBAL_TARGETS) ] --presets [ @babel/preset-env ] ]
-ifeq ($(type),dev)
-	BROWSERIFY += -d
+###--- Shared variables ---###
+# Browser type (browser, but "chrome-mv3" becomes "chrome").
+BROWSER_TYPE = $(browser)
+ifeq ('$(browser)','chrome-mv3')
+  BROWSER_TYPE = chrome
 endif
-KARMA = node_modules/.bin/karma
 
-###--- Variables ---###
+# Output directory for builds.
 BUILD_DIR = build/$(browser)/$(type)
 ifeq ($(browser),test)
-	BUILD_DIR := build/test
+  BUILD_DIR := build/test
 endif
 
-SOURCE_FILES = $(shell find -L shared/ packages/ -type f -not -path "packages/*/node_modules/*" -not -name "*~")
+## All source files that potentially need to be bundled or copied.
```

Comment:
It remains to be seen if being clever with the watch dependencies would help in practice, and it would certainly add complexity + room for mistakes. I figured the simple option was better here, especially if we end up moving to a (much) faster bundler like esbuild.